### PR TITLE
feat(notificationSounds): add notification sounds settings

### PR DESCRIPTION
**Pull request by nickjjang**
AP-368

### CODE REVIEW

**Comment by stavares843** [comment]
https://github.com/Satellite-im/locales/pull/20

**Review by stavares843 — APPROVED** [review]
neat! 🎉

File: components/views/settings/pages/notifications/sounds/Sounds.vue [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<template src="./Sounds.html"></template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { mapState } from 'vuex'
+import { SoundsTypes } from '../../../../../../store/sounds/types'
```
Comment (Jekrimo) [inline components/views/settings/pages/notifications/sounds/Sounds.vue:6]: can we simplify the import from?  '~/whatever' if possible

**Review by Jekrimo — CHANGES_REQUESTED** [review]

File: components/views/settings/pages/notifications/sounds/Sounds.vue [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import Vue from 'vue'
+import { mapState } from 'vuex'
+import { SoundsTypes } from '../../../../../../store/sounds/types'
+
+export default Vue.extend({
+  name: 'NotificationsSettingsSounds',
+  layout: 'sounds',
+  computed: {
+    ...mapState(['sounds']),
+    message: {
```
Comment (WanderingHogan) [inline components/views/settings/pages/notifications/sounds/Sounds.vue:13]: do we need to make these unique, or can we make one generic computed property? Like

toggleSound: {
  get(soundName: string) {
    return this.sounds[soundName]
  }
  ...
}

File: store/sounds/types.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  call = 'call',
+  mute = 'mute',
+  deafen = 'deafen',
+  undeafen = 'undeafen',
+  upload = 'upload',
+  connected = 'connected',
+}
+
+export interface SoundsState {
+  message: Boolean,
```
Comment (WanderingHogan) [inline store/sounds/types.ts:12]: can you use the primitive 'boolean' here instead of the object 'Boolean'

**Review by WanderingHogan — CHANGES_REQUESTED** [review]
couple small things in the comments

File: components/views/settings/pages/notifications/sounds/Sounds.vue (hunk 1 above)
Comment (nickjjang) [reply components/views/settings/pages/notifications/sounds/Sounds.vue]: Fixed it.

File: components/views/settings/pages/notifications/sounds/Sounds.vue (hunk 2 above)
Comment (nickjjang) [reply components/views/settings/pages/notifications/sounds/Sounds.vue]: Fixed it

File: store/sounds/types.ts (hunk 3 above)
Comment (nickjjang) [reply store/sounds/types.ts]: Fixed

**Review by WanderingHogan — APPROVED** [review]
great

**Review by WanderingHogan — APPROVED** [review]
This looks good

**Review by Jekrimo — APPROVED** [review]

**Pull request merged**